tests/berlin/eip2930_access_list/test_tx_intrinsic_gas.py

```diff
@@ -0,0 +1,198 @@
+                AccessList(address=Address(2), storage_keys=[Hash("0x60A7", left_padding=True)]),
+            ],
+            id="access_list_2_address_1_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(address=Address(1), storage_keys=[Hash("0x60A7", left_padding=True)]),
+                AccessList(address=Address(2), storage_keys=[Hash("0x60A8", left_padding=True)]),
+            ],
+            id="access_list_2_address_2_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(
+                    address=Address(1),
+                    storage_keys=[
+                        Hash("0x60A7", left_padding=True),
+                        Hash("0x60A8", left_padding=True),
+                    ],
+                ),
+                AccessList(address=Address(2), storage_keys=[]),
+            ],
+            id="access_list_2_address_2_keys_inversion",
+        ),
+        pytest.param(
+            [
+                AccessList(
+                    address=Address(1),
+                    storage_keys=[
+                        Hash("0xce11", left_padding=True),
+                    ],
+                ),
+                AccessList(
+                    address=Address(2),
+                    storage_keys=[
+                        Hash("0x60A7", left_padding=True),
+                    ],
+                ),
+                *[
+                    AccessList(
+                        address=Address(i),
+                        storage_keys=[
+                            Hash("0x600D", left_padding=True),
+                            Hash("0x0BAD", left_padding=True),
+                            Hash("0x60A7", left_padding=True),
+                            Hash("0xBEEF", left_padding=True),
+                        ],
+                    )
+                    for i in range(3, 13)  # 3 to 12 inclusive (10 entries)
+                ],
+            ],
+            id="access_list_12_address_42_keys",
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "below_intrinsic",
+    [
+        pytest.param(False),
+        pytest.param(True, marks=pytest.mark.exception_test),
+    ],
+)
+def test_tx_intrinsic_gas(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    data: Bytes,
+    access_list: List[AccessList],
+    below_intrinsic: bool,
+):
+    """Transaction intrinsic gas calculation on EIP1559."""
+    tx = Transaction(
+        ty=0x1,
+        chain_id=0x01,
+        sender=pre.fund_eoa(),
+        to=pre.deploy_contract(code=Op.SSTORE(0, Op.ADD(1, 1))),
+        data=data,
+        access_list=access_list,
+        gas_limit=Spec.TX_BASE_INTRINSIC_GAS
+        + compute_data_intrinsic_gas(data)
+        + compute_access_list_intrinsic_gas(access_list)
+        + (-1 if below_intrinsic else 0),
+        error=TransactionException.INTRINSIC_GAS_TOO_LOW if below_intrinsic else None,
+        gas_price=10,
```

Comment:
```suggestion
```
This line can be removed, I'm trying to remove gas price from all tests for them to be compatible with `uv run execute` without modifications.